lib/perl/Genome/Sys.pm

```diff
@@ -1409,6 +1409,12 @@ sub shellcmd {
 
     my ($t1,$t2,$elapsed);
 
+    my @cmdline;
+    if (ref($cmd) and ref($cmd) eq 'ARRAY') {
```

Comment:
Should this `croak` if `set_pipefail` was passed in as an argument?  Hopefully it's obvious to developers that the list version doesn't support piping and so doesn't support the `set_pipefail` option?